docs/api/samples/concept-in-scheme.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+      "dc": "http://purl.org/dc/elements/1.1/",
+      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+      "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+      "skos": "http://www.w3.org/2004/02/skos/core#",
+      "xsd": "http://www.w3.org/2001/XMLSchema#"
+    },
+    "@id": "urn:wbs:47",
+    "@type": "skos:Concept",
+    "skos:broader": [{
+      "@id": "urn:wbs:42"
```

Comment:
Will the linked schemes contain a `skos:prefLabel` here too?

Reply:
I hadn't planned on it, but if you want to have a sort of "Appears in these schemes: <X>, <Y>, <Z>" I'll add it in.

Reply:
That property is for `skos:broader` but I think in general wherever we've got a link off with an `@id` to a concept or concept-scheme we should have at least the `skos:prefLabel`.

Reply:
I'll add a note to the top of the API spec that _ALL_ embedded links _MUST_ provide all available lexical labels (prefLabel, altLabel, hiddenLabel).